### PR TITLE
table add table_log_xx_error()

### PR DESCRIPTION
**Pull request by JackFangXN**
modify some error message about table, code looks better.

### CODE REVIEW

File: src/timedate/timedatectl.c [hunk 1]
```diff
@@ -526,7 +526,7 @@ static int print_ntp_status_info(NTPStatusInfo *i) {
 
         r = table_print(table, NULL);
         if (r < 0)
-                log_error_errno(r, "Failed to show table: %m");
+                table_log_show_error(r);
```
Comment (yuwata) [inline src/timedate/timedatectl.c]: Please also add `return` here.

Comment (JackFangXN) [reply src/timedate/timedatectl.c]: ok，i very agree. now already fix, look better. thks.

File: src/timedate/timedatectl.c [hunk 2]
```diff
@@ -526,7 +526,7 @@ static int print_ntp_status_info(NTPStatusInfo *i) {
 
         r = table_print(table, NULL);
         if (r < 0)
-                log_error_errno(r, "Failed to show table: %m");
+               return table_log_show_error(r);
```
Comment (yuwata) [inline src/timedate/timedatectl.c]: Please use 8ch indentation

Comment (JackFangXN) [reply src/timedate/timedatectl.c]: ok，already fix it, tks，i will be more careful。

Comment (JackFangXN) [reply src/timedate/timedatectl.c]: oh，sorry, i ignore indeed. now all alread fix it. i will be careful next time.

**Pull request merged**